README.md

```diff
@@ -1150,14 +1144,10 @@ Bookmarks
 
 <dl>
 
-[CS50](https://cs50.harvard.edu/2018/fall/)
```

Comment:
Like the previous link for this:

CS50 is a legendary course, it should definitely not be removed, as it is constantly updated each year.

Instead, we can add a link to some of the lectures or their youtube page which hosts the lectures each year:
2023 lectures: https://www.youtube.com/playlist?list=PLhQjrBD2T380F_inVRXMIHCqLaNUd7bN4
YT Page: https://www.youtube.com/channel/UCcabW7890RKJzL968QWEykA